client/store-management-links/index.js

```diff
@@ -0,0 +1,224 @@
+					link: getLinkTypeAndHref( {
+						type: 'wc-settings',
+						tab: 'general',
+					} ),
+					icon: pencil,
+					listItemTag: 'edit-store-details',
+				},
+				{
+					title: __( 'Payments', 'woocommerce-admin' ),
+					link: getLinkTypeAndHref( {
+						type: 'wc-settings',
+						tab: 'checkout',
+					} ),
+					icon: payment,
+					listItemTag: 'payment-settings',
+				},
+				{
+					title: __( 'Tax', 'woocommerce-admin' ),
+					link: getLinkTypeAndHref( {
+						type: 'wc-settings',
+						tab: 'tax',
+					} ),
+					icon: percent,
+					listItemTag: 'tax-settings',
+				},
+				{
+					title: __( 'Shipping', 'woocommerce-admin' ),
+					link: getLinkTypeAndHref( {
+						type: 'wc-settings',
+						tab: 'shipping',
+					} ),
+					icon: shipping,
+					listItemTag: 'shipping-settings',
+				},
+			],
+		},
+	];
+}
+
+export function getLinkTypeAndHref( { path, tab = null, type, href = null } ) {
+	return (
+		{
+			'wc-admin': {
+				href: `admin.php?page=wc-admin&path=%2F${ path }`,
+				linkType: 'wc-admin',
+			},
+			'wp-admin': {
+				href: path,
+				linkType: 'wp-admin',
+			},
+			'wc-settings': {
+				href: `admin.php?page=wc-settings&tab=${ tab }`,
+				linkType: 'wp-admin',
+			},
+		}[ type ] || {
+			href,
+			linkType: 'external',
+		}
+	);
+}
+
+export const generateExtensionLinks = ( links ) => {
+	return links.reduce( ( acc, { icon, href, title } ) => {
+		const url = new URL( href, window.location.href );
+
+		// We do not support extension links that take users away from the host.
+		if ( url.origin === window.location.origin ) {
+			acc.push( {
+				icon,
+				link: {
+					href,
+					linkType: 'extension',
+				},
+				title,
+				listItemTag: 'quick-links-extension-link',
+			} );
+		}
+
+		return acc;
+	}, [] );
+};
+
+export const StoreManagementLinks = () => {
+	const siteUrl = getSetting( 'siteUrl' );
+
+	const extensionQuickLinks = generateExtensionLinks(
+		applyFilters( 'woocommerce_admin_homescreen_quicklinks', [] )
+	);
+
+	const itemCategories = getItemsByCategory( siteUrl );
+
+	const extensionCategory = {
+		title: __( 'Extensions', 'woocommerce-admin' ),
+		items: extensionQuickLinks,
+	};
+
+	const categories = extensionQuickLinks.length
+		? [ ...itemCategories, extensionCategory ]
+		: itemCategories;
+
+	return (
+		<Card size="medium">
+			<CardHeader size="medium">
+				<Text variant="title.small">
+					{ __( 'Store management', 'woocommerce-admin' ) }
+				</Text>
+			</CardHeader>
+			<CardBody
+				size="custom"
+				className="woocommerce-store-management-links__card-body"
+			>
+				{ categories.map( ( category ) => {
+					return (
+						<QuickLinkCategory
+							key={ category.title }
+							title={ category.title }
+						>
+							{ category.items.map(
+								( { icon, listItemTag, title, link } ) => (
+									<QuickLink
+										icon={ icon }
+										key={ listItemTag }
+										title={ title }
+										isExternal={
+											link.linkType === 'external'
```

Comment:
Is there a future use for `linkType`, or is it just to determine the `isExternal` prop? If the latter, what do you think about simplifying this to just be `isExternal` on the link objects instead?

Reply:
@jeffstieler I went back and forth on this and almost removed it. Your validation makes me think we should remove it. Will do.

Reply:
Removed in favor of `isExternal` here: https://github.com/woocommerce/woocommerce-admin/pull/5476/commits/b96c0605b75369f792246b9b1755c5c9a7265259

Reply:
I have opted to go with the `Link` component you suggested in your other comment so have kept the linkType. Changes here: https://github.com/woocommerce/woocommerce-admin/pull/5476/commits/24e2171719d231dcd39305af5ff443a891f509ea